financial-connections/src/main/java/com/stripe/android/financialconnections/features/linkaccountpicker/LinkAccountPickerViewModel.kt

```diff
@@ -270,7 +289,7 @@ internal class LinkAccountPickerViewModel @AssistedInject constructor(
                 // Nothing to log here
             }
         }
-
+        if (payload.acquireConsentOnPrimaryCtaClick) acceptConsent()
```

Comment:
This is for cases where the next pane is not success (because on those consent was acquired via the `share_networked_accounts` endpoint), likely step-up.